Flickr/Sources/FlickrAppVCs/TabsVC.swift

```diff
@@ -6,11 +6,23 @@
 //
 
 import Foundation
+import GRDB
 import UIKit
 
 class TabsVC: UITabBarController {
     private var searchCoordinator: SearchTabCoordinator?
     private var favoritesCoordinator: FavoritesCoordinator?
+    var persistenceManager: PersistenceManager?
+
+    init(persistenceManager: PersistenceManager?) {
+        self.persistenceManager = persistenceManager
+        super.init(nibName: nil, bundle: nil)
+    }
+
```

Comment:
As mentioned earlier, the VC should not hold a reference to the Persistence manager.